spec/shared/constants.rb

```diff
@@ -30,4 +30,113 @@
       "properties"=>[]
     }
   ]
+}.freeze
+
+SAMPLE_INPUT_WITH_PROP =  {
+  "request"=>{
+    "user_info"=>{"user_id"=> "912", "log_user_id"=> "91232"},
+    "use_case"=>"FEED",
+    "properties"=>{
+      "struct"=>{
+        "query"=>{}
+      }
+    }
+  },
+  "full_insertion"=>[
+    {
```

Comment:
If it simplifies writing tests, we don't need as many fields on each properties.